modules/contact/test/tests/pdass_problems/tests

```diff
@@ -0,0 +1,42 @@
+[Tests]
+  design = 'Constraints/ComputeFrictionalForceLMMechanicalContact.md'
+  issues = '#17495'
+  [cylinder_friction]
+    type = 'CSVDiff'
+    csvdiff = 'cylinder_friction_chkfile_friction_0003.csv cylinder_friction_chkfile_x_disp_0003.csv '
+              'cylinder_friction_chkfile_y_disp_0003.csv '
+              'cylinder_friction_chkfile_cont_press_0003.csv'
+    input = 'cylinder_friction.i'
+    rel_err = 1.0e-4
+    min_parallel = 4
+    heavy = true
+    min_ad_size = '50'
+    ad_indexing_type = 'global'
+    requirement = 'Contact module shall solve frictional contact between a cylinder and a plane.'
+  []
+  [ironing]
+    type = 'CSVDiff'
+    csvdiff = 'ironing_chkfile_cont_press_0009.csv ironing_chkfile_friction_0009.csv'
+    input = 'ironing.i'
+    rel_err = 1.0e-4
+    min_parallel = 4
+    heavy = true
+    min_ad_size = '200'
```

Comment:
Very oblique projections appear to be adding a significant number of variable dependencies from the primary surface.

Reply:
I thought very oblique segments are not generated anymore?

Reply:
Extremely oblique segments that cause numerical problems aren't generated. Very oblique segments can make sense, and can span a number of primary surface's lower-d elements, thus a possible reason for the increased required ad size.

Reply:
So this test is just pie in the sky. Never gets tested.